### PR TITLE
cleaning up repo

### LINKED ISSUE #28: Question about compliance with IDY.54 for MC version
Hello
A question for @DT-DawidWroblewski 

Dawid, when I'm reading https://www.gsma.com/identity/wp-content/uploads/2022/12/IDY.54-Mobile-Connect-Verified-MSISDN-Definition-and-Technical-Requirements-1.0.pdf I have:  

> The Resource Server is recommended to expose a service-specific Resource endpoint for supporting the Mobile Connect Verified MSISDN Match service OR premiumInfo endpoint should be used

In our current MC Swagger we have none of them (the endpoint is not specific as same than SIM swap and it is not premiumInfo either).

Perhaps a stupid idea but why not use the path used in the examples of the IDY.54 : POST /connect/mc_vm ?

### LINKED ISSUE #29: sub attribute meaning & cardinality
@DT-DawidWroblewski another point...

`sub `attribute should probably be defined in the API documentation/swagger.
I understood this is an uuid to identified the 'transaction' but this need to be clarified

Another point on this `sub `- as it is not clearly mentioned in IDY.54 is it fair to tag it as mandatory in the response? In Orange implem for example we do not manage it.

Thanks.

### LINKED ISSUE #35: MC solution and guideline alignment
In order to keep track of what we started to discuss in today's meeting I open this issue:

Due to the last decisions that were made in commonalities (based on votes (https://github.com/camaraproject/WorkingGroups/discussions/169) related to this issue(https://github.com/camaraproject/WorkingGroups/issues/157)) the current guideline indicates that the API input and output business data would follow the [lowerCamelCase notation](https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-design-guidelines.md#42-inputoutput-resource-definition)

Currently there are two flavors in Number Verification and Sim Swap APIs. The argument to include MC proposal was to have a solution that is consistent with the available solution in the market AS-IS. But now, we have to align it to the guidelines so it’s not going to be the AS-IS definition. Mobile Connect APIs (proposed in SIM Swap and Number Verification Sub-Projects) parameters follow snake_case notation and now must be changed to lowerCamelCase. 

Maybe it’s the time to reevaluate this part and see what benefits developers can get from having APIs similar to MC but not exactly aligned with them nor with 3GPP/TMF?

### PR DESCRIPTION
**Pull request by DT-DawidWroblewski**
#### What type of PR is this?
* cleanup

#### What this PR does / why we need it:

clean up repo

#### Which issue(s) this PR fixes:

Fixes #35
Fixes #29
Fixes #28

#### Special notes for reviewers:

cleans up repo

#### Changelog input

```
removes MobileConnect definition
creates CAMARA reference to Mobile Connect

```

### CODE REVIEW

File: code/API_definitions/number_verification.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@ info:
   title: Number Verification
   description: |
     Service Enabling Network Function API to verify that the provided **mobile phone number** is the one used in the device. It verifies that the user is using a device with the same *mobile phone number* as it is declared.
+
+    This API derives from the GSMA Mobile Connect Account Takeover Protection specification [Mobile Connect Number Verify](https://www.gsma.com/identity/wp-content/uploads/2022/12/IDY.54-Mobile-Connect-Verified-MSISDN-Definition-and-Technical-Requirements-1.0.pdf).  For more about Mobile Connect, please see [about Mobile Connect](https://mobileconnect.io/).
```
Comment (diegogonmar) [inline code/API_definitions/number_verification.yaml:7]: Yaml should not include this info. This info should be in the .md files within API documentation. Yaml is API spec and has to stick to API Definition

File: documentation/API_documentation/CAMARA/NUMBER_VERIFICATION_API.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Overview
 
 Number Verification API is used to verify that the provided **mobile phone number** is the one used in the device. It verifies that the user is using a device with the same *mobile phone number* as it is declared.
-It also makes it possible for a Service provider to verify the number itself by returning the phone number associated to the authenticated user's access token.
+It also makes it possible for a Service provider to verify the number itself by returning the phone number associated to the authenticated user's access token. This API derives from the GSMA Mobile Connect Account Takeover Protection specification [Mobile Connect Number Verify](https://www.gsma.com/identity/wp-content/uploads/2022/12/IDY.54-Mobile-Connect-Verified-MSISDN-Definition-and-Technical-Requirements-1.0.pdf).  For more about Mobile Connect, please see [about Mobile Connect](https://mobileconnect.io/).
```
[the remark this answers is not in the document]
Comment (diegogonmar) [reply documentation/API_documentation/CAMARA/NUMBER_VERIFICATION_API.md:4]: Also please use references as in the rest of the document, pointing to a references chapter at the end of the doc.

File: documentation/API_documentation/README.MD [hunk 3]
```diff
@@ -4,56 +4,21 @@
...
-CAMARA community agreed to enable Number Verify API in two complementary flavours:
-
-1. Based on CAMARA standardization guidelines, that enables API service on a dedicated endpoint and follows CAMARA AuthN/AuthZ concept. [More details available here](#details)
-2. Based on GSMA Mobile Connect standardized family of Identity APIs, that delivers Verified MSISDN service. [More details available here](#details)
+CAMARA Number Verify API aims to deliver proof of device possession, that is attached to SIM card and MSISDN, using mobile network authentication. This API derives from the GSMA Mobile Connect Account Takeover Protection specification [Mobile Connect Number Verify](https://www.gsma.com/identity/wp-content/uploads/2022/12/IDY.54-Mobile-Connect-Verified-MSISDN-Definition-and-Technical-Requirements-1.0.pdf).  For more about Mobile Connect, please see [about Mobile Connect](https://mobileconnect.io/).
 
 ### Quick start
 
-1. Download one of yaml files.
+1. Download one of yaml file.
```
Comment (diegogonmar) [inline documentation/API_documentation/README.MD:11]:
```suggestion
1. Download yaml file.
```

File: documentation/API_documentation/README.MD [hunk 4]
```diff
@@ -4,56 +4,21 @@
 
 ### Introduction
 
-CAMARA Number Verify API aims to deliver proof of device possession, that is attached to SIM card and MSISDN, using mobile network authentication.
-
-CAMARA community agreed to enable Number Verify API in two complementary flavours:
-
-1. Based on CAMARA standardization guidelines, that enables API service on a dedicated endpoint and follows CAMARA AuthN/AuthZ concept. [More details available here](#details)
-2. Based on GSMA Mobile Connect standardized family of Identity APIs, that delivers Verified MSISDN service. [More details available here](#details)
+CAMARA Number Verify API aims to deliver proof of device possession, that is attached to SIM card and MSISDN, using mobile network authentication. This API derives from the GSMA Mobile Connect Account Takeover Protection specification [Mobile Connect Number Verify](https://www.gsma.com/identity/wp-content/uploads/2022/12/IDY.54-Mobile-Connect-Verified-MSISDN-Definition-and-Technical-Requirements-1.0.pdf).  For more about Mobile Connect, please see [about Mobile Connect](https://mobileconnect.io/).
```
Comment (diegogonmar) [inline documentation/API_documentation/README.MD:7]: Same comment as above regarding proposed re-wording

**Review by diegogonmar — CHANGES_REQUESTED** [review]
Thanks for the cleanup! 
Some additional comments:

- Mobile Connect files should be removed, not moved to archive. It's misleading to a reader to have a reference and then static content, more if archived info is from something not defined here but in a different standard. It would be also problematic to have obsolete information.
- Regarding references, although I proposed alternative wording, I still think that info does not help at all developers and may distract them. In any case reference should be only in .md, not in yaml.

Also some extra cleanup is needed:
This file should be removed --> 
https://github.com/camaraproject/NumberVerification/blob/main/code/API_definitions/README.MD
This folder is no longer needed, adds unneded folder hierarchy level: https://github.com/DT-DawidWroblewski/NumberVerification/tree/main/documentation/API_documentation/CAMARA
(info in the folder should be moved to upper level and empty folder removed)
This folder should be removed --> https://github.com/DT-DawidWroblewski/NumberVerification/tree/main/documentation/API_documentation/assets

**Comment by diegogonmar** [comment]
@DT-DawidWroblewski happy to help, if you're ok I can upload a commit with the extra cleanup and my comments

File: documentation/API_documentation/CAMARA/NUMBER_VERIFICATION_API.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (DT-DawidWroblewski) [reply documentation/API_documentation/CAMARA/NUMBER_VERIFICATION_API.md:4]: This is not true. Camara Number Verification API completely derives from "Number Verify" (www.numberverify.org explicitly stated on various presentaitons) that works on Mobile Connect Verified MSISDN spec.

**Review by bigludo7 — DISMISSED** [review]

File: code/API_definitions/number_verification.yaml (hunk 1 above)
Comment (diegogonmar) [reply code/API_definitions/number_verification.yaml:7]: You're right @bigludo7, I didn't realize that. Suggestion then is to include the reference in the .yaml plus remove rest of documentation, meeting commonalities agreement

**Pull request closed without merging**